test/test_read_all.py

```diff
@@ -43,6 +43,12 @@
 
 
 READABLE_CODECS: Dict[str, Dict[str, List[str]]] = {
+    "jzarr": {
+        "jzarr": ["blosc", "raw", "zlib"],
+        "zarr": ["blosc", "raw", "zlib"],
+        "xtensor_zarr": ["blosc", "raw", "zlib"],
+        "z5py": ["blosc", "raw", "zlib"],
+    },
```

Comment:
```suggestion
    "jzarr": {
        "zarr": ["blosc", "raw", "zlib"],
        "zarr-v3": [],
        "N5": [],
    },
```

The keys here are the file format types, not the other library name (for `zarr` these two are the same so I think it happened to work okay as-is!).